sdk/python/requirements/py3.10-ci-requirements.txt

```diff
@@ -458,7 +460,7 @@ multidict==6.1.0
     # via
     #   aiohttp
     #   yarl
-mypy==1.11.2
+mypy==1.13.0
```

Comment:
revert this one.
I think the issue is due to this change in [1.11.2](https://github.com/python/mypy/blob/master/CHANGELOG.md#inferring-unions-for-conditional-expressions)